lib/fastlane/plugin/wpmreleasetoolkit/calculators/version_calculator.rb

```diff
@@ -0,0 +1,132 @@
+# The `VersionCalculator` class is responsible for performing version calculations and transformations. It can be used
+# as a base class for version calculations that use different versioning schemes. It contains calculation and
+# transformation methods that are shared by all platforms.
+require_relative '../models/app_version'
+
+module Fastlane
+  module Calculators
+    class VersionCalculator
```

Comment:
That makes sense 👍  I think I'll go with `Fastlane::A8CVersioning` instead of `WPMReleaseTookit::Versioning` so there's not an implicit connection only to WP (even though I know it's in the WPMobile org) versus all the other apps.

Reply:
Fixed in https://github.com/wordpress-mobile/release-toolkit/pull/512/commits/37bf610e8a2771aea1097c7aab173b9512c93477 and https://github.com/wordpress-mobile/release-toolkit/pull/512/commits/b178898e08a69e9fee3b6460586aabd6fb5cfc09